python/lsst/ts/watcher/rules/mtm2_tangent_link_temperature.py

```diff
@@ -0,0 +1,164 @@
+        schema_yaml = """
+            $schema: 'http://json-schema.org/draft-07/schema#'
+            description: Configuration for MTM2TangentLinkTemperature rule.
+            type: object
+            properties:
+                buffer:
+                    description: >-
+                        Buffer of the tangent link temperature compared with
+                        the ambient (degree C).
+                    type: number
+                    default: 10.0
+                poll_interval:
+                    description: Time delay between polling updates (second).
+                    type: number
+                    default: 1.0
+
+            required:
+            - buffer
+            - poll_interval
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def setup(self, model) -> None:
+
+        # 106 is the ESS SAL index for the M2 tangent link temperature
+        self._remote_ess = model.remotes[("ESS", 106)]
+
+        self._remote_m2 = model.remotes[("MTM2", 0)]
+
+    def compute_alarm_severity(self) -> AlarmSeverityReasonType:
+        """Compute and set alarm severity and reason.
+
+        Returns
+        -------
+        None, if no change or unknown, or a tuple of two values:
+
+        severity: `lsst.ts.xml.enums.Watcher.AlarmSeverity`
+            The new alarm severity.
+        reason : `str`
+            Detailed reason for the severity, e.g. a string describing
+            what value is out of range, and what the range is.
+            If ``severity`` is ``NONE`` then this value is ignored (but still
+            required) and the old reason is retained until the alarm is reset
+            to ``nominal`` state.
+
+        Notes
+        -----
+        You may return `NoneNoReason` if the alarm state is ``NONE``.
+        """
+
+        assert self._remote_ess is not None
+        assert self._remote_m2 is not None
+
+        if (not self._remote_ess.tel_temperature.has_data) or (
+            not self._remote_m2.tel_temperature.has_data
+        ):
+            return NoneNoReason
+
+        # There are 16 channels in total. Only 6 are used for the M2 tangent
+        # links. Other 10 elements are NaN.
+        temperature_tangent_contain_nan = np.array(
+            self._remote_ess.tel_temperature.get().temperatureItem
+        )
+        temperature_tangent = temperature_tangent_contain_nan[
+            ~np.isnan(temperature_tangent_contain_nan)
+        ]
+
+        temperature_ring = np.array(self._remote_m2.tel_temperature.get().ring)
+
+        return (
+            NoneNoReason
+            if (
+                np.all(
+                    temperature_tangent
+                    < (np.median(temperature_ring) + self.config.buffer)
+                )
+            )
+            else (
+                AlarmSeverity.WARNING,
+                f"Tangent link temperature > ambient by {self.config.buffer} degree C threshold.",
```

Comment:
Consider replacing the">" with text, e.g.:

```
f"Tangent link temperature above ambient threshold of {self.config.buffer} degree C."
```

Reply:
Fixed in the commit: [ce83728](https://github.com/lsst-ts/ts_watcher/pull/83/commits/ce83728096e06c0bb971261e9aa376359b616f92). Thanks!